Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:14.04
+MAINTAINER Samuel Warkentin <s.warkentin@mittwald.de>
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y python python-pip git
+RUN pip install PyYAML tornado --upgrade
+#RUN git clone git://github.com/josh-berry/cronitor.git /opt/cronitor
+RUN mkdir -p /var/log/cronitor
+RUN mkdir -p /etc/cronitor
+
+RUN sudo groupadd -r cronitor
+RUN sudo useradd -r -s /usr/sbin/nologin -g cronitor \
+               -d /opt/cronitor cronitor
+RUN chmod u=rwx,go= /var/log/cronitor
+RUN sudo chown cronitor:cronitor /var/log/cronitor
+
+ADD server.yaml /etc/cronitor/server.yaml
+ADD rules.yaml  /etc/cronitor/rules.yaml
+
+ENV TZ=Europe/Berlin
```

Comment:
Hey,

in Dockerfile you can not pass an local Enviroment.
With docker-compose may be create an .envfile where set the TZ or anything ...

greetings 

Sam

Gruß
Sam